demos/prefsFramework/js/SelfVoicingEnactor.js

```diff
@@ -34,6 +34,9 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         model: {
             value: false
```

Comment:
Following my later comment - I think we can't eliminate "value" from absolutely everywhere - but I think we can from all the enactors.